action.yml

```diff
@@ -79,19 +79,19 @@ inputs:
   nix-build-user-prefix:
     description: The Nix build user prefix (user numbers will be postfixed)
     required: false
-  nix-installer-branch:
-    description: The branch of `nix-installer` to use (conflicts with `nix-installer-tag`, `nix-installer-revision`, `nix-installer-pr`)
+  source-branch:
+    description: The branch of `nix-installer` to use (conflicts with `source-tag`, `source-revision`, `source-pr`)
     required: false
-  nix-installer-pr:
-    description: The PR of `nix-installer` to use (conflicts with `nix-installer-tag`, `nix-installer-revision`, `nix-installer-branch`)
+  source-pr:
+    description: The PR of `nix-installer` to use (conflicts with `source-tag`, `source-revision`, `source-branch`)
     required: false
-  nix-installer-revision:
-    description: The revision of `nix-installer` to use (conflicts with `nix-installer-tag`, `nix-installer-branch`, `nix-installer-pr`)
+  source-revision:
+    description: The revision of `nix-installer` to use (conflicts with `source-tag`, `source-branch`, `source-pr`)
     required: false
-  nix-installer-tag:
-    description: The tag of `nix-installer` to use (conflicts with `nix-installer-revision`, `nix-installer-branch`, `nix-installer-pr`)
+  source-tag:
+    description: The tag of `nix-installer` to use (conflicts with `source-revision`, `source-branch`, `source-pr`)
     required: false
```

Comment:
I realize these are renames of existing functionality, but it seems like `source-url` more or less can encompass all of these usages - preusmably you can "just" use a URL that points to the branch/pr/rev/tag?

Reply:
We could, but I've seen time and time again folks (internal & external) mess up the structure, so I think having these sorts of helpers makes it a nicer experience for testing PRs and whatnot.